MATLAB/DFA/src/EDA/exper_data.m

```diff
@@ -0,0 +1,228 @@
+
+Tabl_std_HCT116_GOT1 = Tabl_std_HCT116(:, {'metabolite', 'HCT116_shGOT1_no_dox_1','HCT116_shGOT1_2d_dox_1', 'HCT116_shGOT1_4d_dox_1', 'HCT116_shGOT1_no_dox_media_1', 'HCT116_shGOT1_2d_dox_media_1', 'HCT116_shGOT1_4d_dox_media_1'});
+Tabl_std_HCT116_NT = Tabl_std_HCT116(:, {'metabolite', 'HCT116_shNT_no_dox_1','HCT116_shNT_2d_dox_1', 'HCT116_shNT_4d_dox_1', 'HCT116_shNT_no_dox_media_1', 'HCT116_shNT_2d_dox_media_1', 'HCT116_shNT_4d_dox_media_1'});
+
+% CELL LINE: TU9802
+Tabl_avg_TU8902_GOT1 = Tabl_average_TU8902(:, {'metabolite', 'TU8902_shGOT1_no_dox_1','TU8902_shGOT1_2d_dox_1', 'TU8902_shGOT1_4d_dox_1', 'TU8902_shGOT1_no_dox_media_1', 'TU8902_shGOT1_2d_dox_media_1', 'TU8902_shGOT1_4d_dox_media_1'});
+Tabl_avg_TU8902_NT = Tabl_average_TU8902(:, {'metabolite', 'TU8902_shNT_no_dox_1','TU8902_shNT_2d_dox_1', 'TU8902_shNT_4d_dox_1', 'TU8902_shNT_no_dox_media_1', 'TU8902_shNT_2d_dox_media_1', 'TU8902_shNT_4d_dox_media_1'});
+
+Tabl_std_TU8902_GOT1 = Tabl_std_TU8902(:, {'metabolite', 'TU8902_shGOT1_no_dox_1','TU8902_shGOT1_2d_dox_1', 'TU8902_shGOT1_4d_dox_1', 'TU8902_shGOT1_no_dox_media_1', 'TU8902_shGOT1_2d_dox_media_1', 'TU8902_shGOT1_4d_dox_media_1'});
+Tabl_std_TU8902_NT = Tabl_std_TU8902(:, {'metabolite', 'TU8902_shNT_no_dox_1','TU8902_shNT_2d_dox_1', 'TU8902_shNT_4d_dox_1', 'TU8902_shNT_no_dox_media_1', 'TU8902_shNT_2d_dox_media_1', 'TU8902_shNT_4d_dox_media_1'});
+
+%% Create Numeric Arrays of all Tables
+% Cell Line: TU8902
+%all conditions
+mean_metabolomic_array_TU8902 = table2array(Tabl_average_TU8902(:,2:end));
+std_metabolomic_array_TU8902 = table2array(Tabl_std_TU8902(:,2:end));
+
+%GOT1 and NT
+avg_TU8902_GOT1 = table2array(Tabl_avg_TU8902_GOT1(:, 2:end));
+avg_TU8902_NT = table2array(Tabl_avg_TU8902_NT(:, 2:end));
+
+std_TU8902_GOT1 = table2array(Tabl_std_TU8902_GOT1(:, 2:end));
+std_TU8902_NT = table2array(Tabl_std_TU8902_NT(:, 2:end));
+
+% Cell Line: HCT116
+%all conditions
+mean_metabolomic_array_HCT116 = table2array(Tabl_average_HCT116(:,2:end));
+std_metabolomic_array_HCT116 = table2array(Tabl_std_HCT116(:,2:end));
+
+%GOT1 and NT
+avg_HCT116_GOT1 = table2array(Tabl_avg_HCT116_GOT1(:, 2:end));
+avg_HCT116_NT = table2array(Tabl_avg_HCT116_NT(:, 2:end));
+
+std_HCT116_GOT1 = table2array(Tabl_std_HCT116_GOT1(:, 2:end));
+std_HCT116_NT = table2array(Tabl_std_HCT116_NT(:, 2:end));
+
+%% Create the coefficient of variation, CV 
+%CV matrix for all combinations
+
+CV_TU8902 = std_metabolomic_array_TU8902 ./ mean_metabolomic_array_TU8902;
+CV_TU8902_GOT1 = std_TU8902_GOT1 ./ avg_TU8902_GOT1;
+CV_TU8902_NT = std_TU8902_NT ./ avg_TU8902_NT;
+
+CV_HCT116 = std_metabolomic_array_HCT116 ./ mean_metabolomic_array_HCT116;
+CV_HCT116_GOT1 = std_HCT116_GOT1 ./ avg_HCT116_GOT1;
+CV_HCT116_NT = std_HCT116_NT ./ avg_HCT116_NT;
+
+%% DFA Filter 1: CV thresholds + NaN: 0.1

```

Comment:
This bit here can be shortened using the following pseudocode:
cv = [0.1, 0.3, 0.6]
for i = 1:length(cv)
  Do all operations
  Store result in cell array
end